src/guide/built-ins/transition-demos/NestedTransitions.vue

```diff
@@ -26,6 +26,7 @@ const show = ref(true)
 
 .transition-demo-inner {
   background: #ccc;
+  color: rgb(33, 53, 72);
```

Comment:
```suggestion
  color: rgb(33, 53, 71);
```

En el [commit original](https://github.com/vuejs/docs/commit/64d58ad9a14b302b292b669a033328e330f68b13) el ultimo valor es `71`